__tests__/App.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import 'react-native';
+
+import { render } from '@testing-library/react-native';
+
+import App from '../App';
+
+describe('App', () => {
+  it('renders correctly', () => {
+    const app = render(<App />);
+    expect(app).toBeTruthy();
+  });
+
+  it('elements should be accessible', () => {
+    const { getByText, getByA11yState } = render(<App />);
+    const text = getByText('Welcome Screen');
+    const button = getByA11yState({ disabled: false });
```

Comment:
Getting sth by it's state seems weird 🤔. Does this engine has a `getByRole` or something similar?

Reply:
I did this just for demonstration purposes. I'll change it to use `getByRole`.